src/README.rst

```diff
@@ -19,6 +19,7 @@ Change Log
 Unreleased
 -----
 - Add application health policies parameter for config upgrade, which are available in Service Fabric runtime 6.3 (#92)
+- Add command to see the current version and target azure-servicefabric version (#26)
```

Comment:
We should check to see if `knack` supports something like `sfctl --version`. To me that's more intuitive and the standard over running `sfctl version`

Reply:
Knack actually does have built in support for displaying a version through --version/ -v